### PR TITLE
goatools support py35 and py36

### PR DESCRIPTION
**Pull request by yyminmax**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/goatools/meta.yaml [hunk 1]
```diff
@@ -5,40 +5,37 @@ about:
...
+  md5: f2ab989ec9c4acdd80504b263c3b3188
 
 build:
-  skip: True # [not py27]
   number: 0
 
 requirements:
   build:
     - python
-    - setuptools
```
Comment (mbargull) [inline recipes/goatools/meta.yaml]: `setuptools` is a build requirement. Actually only
```yaml
requirements:
  build:
    - python
    - setuptools
```
is needed for the build.

File: recipes/goatools/build.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-$PYTHON setup.py install
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
 
 binaries="\
```
Comment (mbargull) [inline recipes/goatools/build.sh]: Those scripts should be installed by `setuptools`, hence `build.sh` can be reduced to only
```
 #!/bin/bash
 
$PYTHON setup.py install --single-version-externally-managed --record=record.txt
 ```

**Comment by mbargull** [comment]
I added some commits, mainly to use `setuptools` for the build and also to exclude the test data from the package as they don't seem to be required for normal operation (but make the package require >100 times more storage space). I hope you don't mind, that I added those changes.

Otherwise your changes look good to me, thanks!

**Comment by yyminmax** [comment]
Thanks for your changes. It looks great.

**Comment by bgruening** [comment]
Cool, thanks a lot!

**Pull request merged**